source/beamline-configuration.rst

```diff
@@ -26,29 +23,48 @@ quick start page, they can be invoked by typing:
 
     ipython --profile=collection
 
+To refresh a script that has been edited, restart IPython or re-execute the
+script using ``%run -i ~/.ipython/profile_collection/startup/<FILENAME>``.
+Do this for any file(s) that have been updated. (There is no way to run
+all files at once.)
+
 Example Configuration File
 --------------------------
 
 This is a example IPython profile startup file.::
 
-    from bluesky.standard_config import *  # get all of the above for free
-
-    RE = gs.RE  # convenient alias
-    RE.md['beamline_id'] = 'YOUR_BEAMLINE_HERE'
-
-    import ophyd
-    from ophyd import *
-    from ophyd.commands import setup_ophyd
-    from bluesky.plans import *
-    from bluesky.callbacks import *
-
-    # Set up ophyd for use (this is required)
+    # Make ophyd listen to pyepics.
+    from ophyd import setup_ophyd
     setup_ophyd()
 
+    # Subscribe metadatastore to documents.
+    # If this is removed, data is not saved to metadatastore.
+    from bluesky.register_mds import register_mds
+    RE = gs.RE  # convenience alias
+    register_mds(RE)
+
     # Import matplotlib and put it in interactive mode.
     import matplotlib.pyplot as plt
     plt.ion()
 
+    # Make plots update live while scans run.
+    from bluesky.qt_kicker import install_qt_kicker
```

Comment:
This PR happened before I decided to deprecate qt_kicker today. This particular line has since been updated by a sneaky push to master. :imp: